java-agent/instrumentation/netty-4.0/src/test/groovy/Netty40ClientTest.groovy

```diff
@@ -71,6 +71,7 @@ class Netty40ClientTest extends HttpClientTest<NettyHttpClientDecorator> {
     then:
     def ex = thrown(Exception)
     def thrownException = ex instanceof ExecutionException ? ex.cause : ex
+    println "Exception: $ex"
```

Comment:
Just checking if this was an artifact of debugging, or intentionally part of PR, I'm ok either way

Reply:
This is leftover debugging. I'll get rid of it.